src/1-raw-data/gcs/gcs.go

```diff
@@ -25,51 +25,60 @@ import (
 	"cloud.google.com/go/storage"
 	"context"
 	"fmt"
+	"github.com/googleapis/google-cloud-go-testing/storage/stiface"
 	"google.golang.org/api/iterator"
 	"io"
 	"log"
 	"net/http"
 )
 
+type GCSConnection interface {
```

Comment:
Given that this package is called `gcs`, consider dropping the `GCS` parts of the type names here (go style is somewhat pedantic about what it calls "stuttering"). So this type would be just `Connection` and from the main, `gcs.Connection` which works well I think...